src/mito2/src/read/seq_scan.rs

```diff
@@ -65,7 +65,11 @@ impl SeqScan {
     /// Creates a new [SeqScan].
     pub(crate) fn new(input: ScanInput) -> Self {
         let parallelism = input.parallelism.parallelism.max(1);
-        let properties = ScannerProperties::new_with_partitions(parallelism);
+        let properties = ScannerProperties {
+            partitions: vec![vec![]; parallelism],
+            append_mode: input.append_mode,
+            total_rows: input.total_rows(),
+        };
```

Comment:
non-blocking: Maybe we can add two functions `with_append_mode()` and `with_total_rows()` so we can avoid repeating `vec![vec![]; parallelism]`.

Reply:
fixed in 92e5295